IntegrationTests.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3-alpine3.8
+
+RUN apk add --no-cache gcc build-base libffi-dev openssl-dev
+COPY requirements-integration-tests.txt /tmp
```

Comment:
The file is so small we could consider putting the items directly into this file.
Less files inside the repository and less layers in the image

Reply:
That is a really bad idea. If you care for image layers, just squash the image when building on the CI. For local development more layers are always better, because you utilise the caching of the docker daemon.